views/main.erb

```diff
@@ -59,7 +59,7 @@ function handleSubmit()
 
 document.getElementById('get').onclick = function() {
     xmlHttp = new XMLHttpRequest();
-    xmlHttp.open( "GET", "http://"+window.location.hostname+":"+window.location.port+"/keys/"+document.getElementById("key").value, false );
+    xmlHttp.open( "GET", "https://"+window.location.hostname+":"+window.location.port+"/keys/"+document.getElementById("key").value, false );
```

Comment:
Like it. I will fix tonight

Sent from my phone please excuse typos,  grammar, and directness
On Jun 1, 2015 11:52 PM, "Jordan Liggitt" <notifications@github.com> wrote:

> In views/main.erb
> <https://github.com/openshift/ruby-hello-world/pull/39#discussion_r31494327>
> :
>
> > @@ -59,7 +59,7 @@ function handleSubmit()
> >
> >  document.getElementById('get').onclick = function() {
> >      xmlHttp = new XMLHttpRequest();
> > -    xmlHttp.open( "GET", "http://"+window.location.hostname+":"+window.location.port+"/keys/"+document.getElementById("key").value, false );
> > +    xmlHttp.open( "GET", "https://"+window.location.hostname+":"+window.location.port+"/keys/"+document.getElementById("key").value, false );
>
> this will break when served over http. use
> window.location.protocol+"//"+window.location.hostname...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/ruby-hello-world/pull/39/files#r31494327>.
>